### PR TITLE
Add RATE_LIMIT_BYPASS_TOKEN to link-checker-api

### PR DESCRIPTION
**Pull request by SebAshton**
This PR adds a token to the `link-checker-api` `ENV` to bypass the rate limiter within nginx

### CODE REVIEW

**Comment by SebAshton** [comment]
Thanks @issyl0, I've made a request to 2ndLine to get this added.

**Comment by surminus** [comment]
This will create an environment variable that the application can use, but a rate-limit token will need to be added to NGINX to allow pass through.

You will need to add the token in `router::nginx::rate_limit_tokens` in govuk-secrets. You will see similar ones added.

File: modules/govuk/manifests/apps/link_checker_api.pp [hunk 1]
```diff
@@ -49,6 +49,10 @@
 #   start.
 #   Default: undef
 #
+# [*rate_limit_bypass_token*]
```
Comment (thomasleese) [inline modules/govuk/manifests/apps/link_checker_api.pp]: Can we rename this to `rate_limit_token` for consistency with the other apps?

Comment (SebAshton) [reply modules/govuk/manifests/apps/link_checker_api.pp]: Sure @thomasleese, have updated.

**Comment by thomasleese** [comment]
Secrets have been merged in now.

**Pull request merged**